src/main/kotlin/graphics/scenery/primitives/LineBetweenNodes.kt

```diff
@@ -0,0 +1,24 @@
+package graphics.scenery.primitives
+
+import graphics.scenery.attribute.spatial.Spatial
+import org.joml.Vector3f
+
+/**
+ * A line which always goes from one node to another.
+ */
+class LineBetweenNodes(var from: Spatial, var to: Spatial, transparent: Boolean = false, simple: Boolean = false) :
```

Comment:
Not entirely sure this is necessary. You could make a unit line, and then use `Node.orientBetweenPoints`.

Reply:
Switched to `Node.orientBetweenPoints`. 
The From and To spatials need to be saved somewhere for the update to work. IMO it's the moste clean solution to save them in an extra class.